### ISSUE TITLE
Readme basic documentation

### ISSUE THREAD

**Issue by tlaurion** [fepitre/qubes-mgmt-salt-qubes-server#1]
This seems really promising.

Can you document a bit the actual status of the project, missing pieces and help needed?

Thanks for your work!!!

**Comment by fepitre** [comment]
Of course, it's in progress. I'm currently finishing GuiVM related stuff then I need to setup properly dependencies in QubesOS-contrib, build packages and do the doc.

Btw, sorry for the late reply.

**Comment by tlaurion** [comment]
Close with reference to doc @fepitre ?
